website/cue/reference/components/sources/syslog.cue

```diff
@@ -97,6 +97,22 @@ components: sources: syslog: {
 				examples: ["/path/to/socket"]
 			}
 		}
+		socket_file_mode: {
```

Comment:
```suggestion
		socket_file_mode: sources.socket.configuration.socket_file_mode
```

This'll help them stay in sync by just re-using the socket config definition here.